configure.ac

```diff
@@ -226,6 +226,18 @@ AC_MSG_CHECKING([whether CC supports -Wsign-conversion])
 AC_COMPILE_IFELSE([AC_LANG_PROGRAM([])], [AC_MSG_RESULT([yes]); CPPUTEST_CWARNINGFLAGS="${CPPUTEST_CWARNINGFLAGS} -Wsign-conversion"], [AC_MSG_RESULT([no])])
 CFLAGS="$saved_cflags"
 
+# Flag -Wreserved-id-macro
+CFLAGS="-Werror -Wreserved-id-macro"
+AC_MSG_CHECKING([whether CC support -Wreserved-id-macro])
+AC_COMPILE_IFELSE([AC_LANG_PROGRAM([])], [AC_MSG_RESULT([yes]); CPPUTEST_CWARNINGFLAGS="${CPPUTEST_CWARNINGFLAGS} -Wreserved-id-macro"], [AC_MSG_RESULT([no])])
+CFLAGS="$saved_cflags"
+
+# Flag -Wno-keyword-macro.
```

Comment:
typo in comment.

Do we need to enable these explicitly? Aren't they included in some of the warning sets like -Wall ?

Reply:
Good catch. I've switched to the broader, non-default `-Wreserved-identifier`.